docs/en/observability/synthetics-role-write.asciidoc

```diff
@@ -83,8 +83,12 @@ Additional privileges will depend on the factors below.
 [discrete]
 == If using Private Locations
 
-If the user should be able to create and update monitors hosted on
-<<synthetics-private-location,Private Locations>>, add the following privileges:
+No additional permissions are needed if the user should be able to create and update monitors hosted on
+<<synthetics-private-location,Private Locations>>. However when the agent policy is initially created within {fleet}, the following privileges are required:
+
+The user who initially sets up a new Private Location needs additional privileges. Users who create or update monitors hosted on that Private Location do not need any additional permissions.
+
+The user who is setting up a new Private Location will need the following privileges when creating the agent policy in {fleet}:
```

Comment:
Did you mean to delete the first paragraph? It currently [looks like this](https://observability-docs_3008.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-role-write.html#_if_using_private_locations): 

<img width="857" alt="Screenshot 2023-06-22 at 12 50 18 PM" src="https://github.com/elastic/observability-docs/assets/10479155/a81a37d8-d1e8-4d1c-9ee8-d1c8b3edce9e">

```suggestion
The user who initially sets up a new Private Location needs additional privileges. Users who create or update monitors hosted on that Private Location do not need any additional permissions.

The user who is setting up a new Private Location will need the following privileges when creating the agent policy in {fleet}:
```